src/main/java/fi/csc/chipster/jobhistory/JobHistoryModel.java

```diff
@@ -0,0 +1,100 @@
+package fi.csc.chipster.jobhistory;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.xml.bind.annotation.XmlRootElement;
+
+
+
+@Entity
+@XmlRootElement
+public class JobHistoryModel {
+	//UserName
+	//tool name
+	//comp
+	//time duration
+	//End time
+	//Output
+	//Status
+	 
+	//Filter needed 
+	//Username
+	//tool name
+	//comp
+	//time duration
+	//End time
+	//Output
+	//Status
+	
+	@Id
+	@Column
+	private UUID jobId;
```

Comment:
if I remember correctly, this was converted to a string column in SQL. See other objects for an example about an annotation which makes sure that the H2 saves this as a more efficient UUID type.